src/FastBoard.h

```diff
@@ -40,7 +40,7 @@ class FastBoard {
     /*
         largest board supported
     */
-    static constexpr int MAXBOARDSIZE = 19;
+    static constexpr int MAXBOARDSIZE = BOARD_SIZE;
```

Comment:
This no longer has any reason of existence.

Reply:
Maybe defining something here or elsewhere that is BOARD_SIZE * BOARD_SIZE (SQUARE_COUNT?) or so would save a lot of redundancy.

Square count sounds a bit odd because go is played on the intersections, but I can't think of anything better right now.

Reply:
Do you mean I can simply remove the line, or should I keep it with 19?

Reply:
As for the constant for BOARD_SIZE * BOARD_SIZE, maybe BOARD_SQUARES as @sethtroisi proposed, or something like INTERSECTION_COUNT?

Reply:
Let's take BOARD_SQUARES. The line above can go as MAXBOARDSIZE==BOARD_SIZE.